tests/test_parser.py

```diff
@@ -225,3 +225,12 @@ def test_e_grammer_error_at_eof():
         assert str(e) == 'Grammer error at EOF'
     else:
         raise
+
+
+def test_e_use_thrift_reserved_keywords():
+    try:
```

Comment:
This test better write with `pytest.raises`

https://pytest.org/latest/assert.html#assertions-about-expected-exceptions